### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by mprasoonquartic**
Move docs from md to rst

### CODE REVIEW

**Review by scriperdjq — COMMENTED** [review]
why is it raised against master?

**Comment by mprasoonquartic** [comment]
> why is it raised against master?

We are keeping the `master` branch as the main branch in SDK instead of `develop`. So, we can remove `develop` from usage, based upon discussion with @pr4n

**Comment by scriperdjq** [comment]
Not sure why we decided that. This means every individual change requires a bump in production package. We are following develop branch strategy for other repo for multiple other reasons that were earlier discussed (https://nvie.com/posts/a-successful-git-branching-model/). None of that is applicable here?

**Review by pr4n — COMMENTED** [review]
No change in README?

**Review by pr4n — CHANGES_REQUESTED** [review]
Without the presence of updates README, I am unable to generate documentation locally. Hence I can not proceed.

**Comment by mprasoonquartic** [comment]
> Without the presence of updates README, I am unable to generate documentation locally. Hence I can not proceed.

Yes. I have updated the README

**Review by DineshQuartic — DISMISSED** [review]

**Review by vibhuQuartic — APPROVED** [review]

File: docs/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+=========================
+Quartic SDK documentation
+=========================
+
+Quartic SDK is Quartic.ai's external software development kit (SDK) which
+allows users to use the assets, tags, and other intelligence of the Quartic AI Platform externally. Using the Quartic SDK, third party developers who
+to build custom applications. uartic SDK supports two kinds of authentication: Basic Authentication
```
Comment (pr4n) [inline docs/source/index.rst:7]: Spelling of Quartic

Comment (pr4n) [inline docs/source/index.rst:7]: doesnt make sense 
> Using the Quartic SDK, third party developers who
 to build custom applications

File: docs/source/model_creation.rst [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+ModelABC
+-----------
+
+ModelABC is a base class for all the ML models that can be deployed to
+the Platform. Users must extend this class and implement the predict
+method to make the ML model compatible to deploy in the Quartic AI
+Platform. The available methods are as follows:
+
+init
```
Comment (pr4n) [inline docs/source/model_creation.rst:15]: `.init` ?

File: docs/source/model_creation.rst [hunk 3]
```diff
@@ -0,0 +1,154 @@
+Model Creation
+==============
+
+This article explains how to create, wrap, and deploy models to the
+Quartic AI Platform.
+
+ModelABC
```
Comment (pr4n) [inline docs/source/model_creation.rst:7]: I had a discusion with @DineshQuartic to update this to QuarticBaseModel. Maybe thats in another PR? Will this need to be updated when that PR Merges?

File: docs/source/model_creation.rst (hunk 2 above)
Comment (pr4n) [reply docs/source/model_creation.rst:15]: This is also at other places. APIClient's init.

**Review by pr4n — DISMISSED** [review]
Minor changes. LGTM otherwise

File: docs/source/model_creation.rst (hunk 3 above)
Comment (DineshQuartic) [reply docs/source/model_creation.rst:7]: I think few of lines were missed in doc update, updating same with mayank

File: docs/source/model_creation.rst (hunk 2 above)
Comment (DineshQuartic) [reply docs/source/model_creation.rst:15]: init in here is useful for client when he call `super().__init__(**params)` while implementing `BaseQuarticModel`. I think this should be `__init__` not init

File: docs/source/index.rst (hunk 1 above)
Comment (mprasoonquartic) [reply docs/source/index.rst]: resolved

Comment (mprasoonquartic) [reply docs/source/index.rst]: resolved

File: docs/source/model_creation.rst (hunk 2 above)
Comment (mprasoonquartic) [reply docs/source/model_creation.rst]: removed init from other places

File: docs/source/model_creation.rst [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+This is a private method used to save the model to the Quartic AI
+Platform.
+
+.. raw:: html
+
+   <div class="note">
+
+Note: While implementing an instance of BaseQuarticModel, users must call
+super().\ **init**\  with above parameters.
```
Comment (DineshQuartic) [inline docs/source/model_creation.rst:49]: I think this note should be above .save, this note is related to init method

**Review by pr4n — APPROVED** [review]
Okay. I am good at the moment. We will anyways update the doc to fetch it from the generated code comments later. So its lgtm

**Review by DineshQuartic — APPROVED** [review]

**Pull request merged**